firebase-functions/src/main/java/com/google/firebase/functions/FirebaseFunctions.kt

```diff
@@ -227,8 +227,8 @@ internal constructor(
     val encoded = serializer.encode(data)
     body["data"] = encoded
     val bodyJSON = JSONObject(body)
-    val contentType = MediaType.parse("application/json")
-    val requestBody = RequestBody.create(contentType, bodyJSON.toString())
+    val contentType = "application/json".toMediaTypeOrNull()
+    val requestBody = bodyJSON.toString().toRequestBody(contentType)
```

Comment:
I'm wondering whether these changes would cause binary incompatibility for devs who have an explicit dependency on okhttp 3.x in their apps.